webapp-django/crashstats/documentation/jinja2/documentation/products.html

```diff
@@ -12,55 +12,44 @@ <h1 id="policy">Products on Crash Stats</h1>
     </p>
     <p>
         Incoming crash reports have a <code>ProductName</code> key which
-        specifies which product the crash report is for. For example, incoming
-        crash reports for Firefox desktop have <code>Firefox</code> as the
-        key. The crash reporter sets this using the value of
-        <code>MOZ_APP_BASENAME</code> in <code>browser/confvars.sh</code>.
+        specifies which product the crash report is for.
```

Comment:
suggestions:
which product is described by the crash report?
with which product the crash report should be grouped?
from which product the crash report originated?